kpatch-build/kpatch-gcc

```diff
@@ -15,7 +15,10 @@ if [[ "$TOOLCHAINCMD" = "gcc" ]] ; then
 	while [ "$#" -gt 0 ]; do
 		if [ "$1" = "-o" ]; then
 			obj=$2
-			[[ $2 = */.tmp_*.o ]] && obj=${2/.tmp_/}
+			# skip copying the original .o files, when .tmp_mc_*.o
+			# is passed from recordmcount.pl.
+			[[ $2 = */.tmp_mc_*.o ]] && break;
+			[[ $obj = */.tmp_*.o ]] && obj=${2/.tmp_/}
```

Comment:
The second line is the original code (interchanged the $2 and $obj), which is required for append the changed objects file to ```$KPATCH_GCC_TEMPDIR/changed_objs```. Sure, will change the usage of $2.